test/util/velero/install.go

```diff
@@ -112,27 +113,31 @@ func VeleroInstall(ctx context.Context, veleroCfg *VeleroConfig, isStandbyCluste
 	if err != nil {
 		return errors.WithMessagef(err, "Failed to get Velero InstallOptions for plugin provider %s", veleroCfg.ObjectStoreProvider)
 	}
-	veleroInstallOptions.UseVolumeSnapshots = veleroCfg.UseVolumeSnapshots
-	if !veleroCfg.UseRestic {
-		veleroInstallOptions.UseNodeAgent = veleroCfg.UseNodeAgent
-	}
-	veleroInstallOptions.UseRestic = veleroCfg.UseRestic
-	veleroInstallOptions.Image = veleroCfg.VeleroImage
-	veleroInstallOptions.Namespace = veleroCfg.VeleroNamespace
-	veleroInstallOptions.UploaderType = veleroCfg.UploaderType
-	GCFrequency, _ := time.ParseDuration(veleroCfg.GCFrequency)
-	veleroInstallOptions.GarbageCollectionFrequency = GCFrequency
-	veleroInstallOptions.PodVolumeOperationTimeout = veleroCfg.PodVolumeOperationTimeout
-	veleroInstallOptions.NodeAgentPodCPULimit = veleroCfg.NodeAgentPodCPULimit
-	veleroInstallOptions.NodeAgentPodCPURequest = veleroCfg.NodeAgentPodCPURequest
-	veleroInstallOptions.NodeAgentPodMemLimit = veleroCfg.NodeAgentPodMemLimit
-	veleroInstallOptions.NodeAgentPodMemRequest = veleroCfg.NodeAgentPodMemRequest
-	veleroInstallOptions.VeleroPodCPULimit = veleroCfg.VeleroPodCPULimit
-	veleroInstallOptions.VeleroPodCPURequest = veleroCfg.VeleroPodCPURequest
-	veleroInstallOptions.VeleroPodMemLimit = veleroCfg.VeleroPodMemLimit
-	veleroInstallOptions.VeleroPodMemRequest = veleroCfg.VeleroPodMemRequest
-	veleroInstallOptions.DisableInformerCache = veleroCfg.DisableInformerCache
 
+	if veleroCfg.CloudProvider == "aws" && veleroInstallOptions.ServiceAccountName != "" {
```

Comment:
I would add a comment here that this _if_ block is for IRSA.

Reply:
Good point! I also added a checkpoint for `EKSPolicyARN` parameter existence since it's a required parameter for IRSA test.